templates/sls-app-template-go/functions/function1/main.go

```diff
@@ -36,7 +36,7 @@ func Handler(ctx context.Context, event map[string]interface{}) (Response, error
 	 logger.VERBOSE("Generally speaking, most lines logged by your application should be written as verbose.");
 	 logger.DEBUG("Detailed information on the flow through the system.");
 	*/
-	logger.INFO("Interesting runtime events (Eg. connection established, data fetched etc.)")
+	logger.INFO("Sample log for function1")
```

Comment:
keep the log statement. It gives context to the user

Reply:
Done